source/navitiacommon/navitiacommon/default_values.py

```diff
@@ -92,6 +92,9 @@
 # night bus filtering parameter
 night_bus_filter_base_factor = 60*60  # seconds
 
+#priority value to be used to choose un instance kraken among valid instances
```

Comment:
Maybe: 

priority value to be used to choose a kraken instance among valid instances. The greater it is, the more the instance will be chosen